### PR TITLE
Add site name to exported data [AQCSV]

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**
Adds site name to exported data (AQCSV)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [PLAT-1196](https://airqoteam.atlassian.net/browse/PLAT-1196)

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

**Comment by 123MwanjeMike** [comment]
Changes in this PR can be previewed from [here](https://plat-1196-aqcsv-data-export-analytics-preview-w7kzhvlewq-ew.a.run.app)

File: src/analytics/api/utils/data_formatters.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def format_to_aqcsv(data: list, pollutants: list, frequency: str) -> list:
             "timestamp": "datetime",
             "latitude": "lat",
             "longitude": "lon",
-            "site_id": "site",
+            "name": "site_name",
```
Comment (Baalmart) [inline src/analytics/api/utils/data_formatters.py:26]: Could we make this more descriptive? perhaps `site_name`? @NoahNsimbe

**Review by Baalmart — COMMENTED** [review]
All LGTM, just one small suggestion @NoahNsimbe

File: src/analytics/api/utils/data_formatters.py (hunk 1 above)
Comment (NoahNsimbe) [reply src/analytics/api/utils/data_formatters.py:26]: It is already `site_name` @Baalmart

**Pull request merged**